website/client/components/tasks/taskModal.vue

```diff
@@ -896,6 +907,8 @@ export default {
         this.task.group.assignedUsers = this.assignedMembers;
```

Comment:
Let's make a todo here to flatten out group tasks and add correct permissions so we don't have to do this hack. We will want to do this soon.